File: tests/test06.pl
#!/usr/bin/perl -w
# Test for Subset 3 - chomp, split, join, ARGV

$joined = join(' ', @ARGV);
@split = split(' ', $joined);
for $argv (@ARGV) {
	chomp $argv;
	print "argv: $argv\n";
	print "$joined\n";
}

for $l (@split) {
	print "$l\n";
}

File: tests/test06.py
#!/usr/bin/python2.7 -u
import sys
# Test for Subset 3 - chomp, split, join, ARGV

joined = ' '.join(sys.argv[1:])
split = joined.split(' ')
for argv in sys.argv[1:]:
	argv = argv.rstrip()
	print "argv: %s" % (argv)
	print "%s" % (joined)

for l in split:
	print "%s" % (l)
